src/app/index.tsx

```diff
@@ -1,10 +1,21 @@
-import React, { Component } from 'react';
-import "@patternfly/react-core/dist/styles/base.css";
+import * as React from 'react';
+import '@patternfly/react-core/dist/styles/base.css';
```

Comment:
Can we move the page code out of index.tsx and put it in there own components?  Beo yice to see it where there is a AppLayout which contains the Page AppToolbar for the main toolbar,AppHeader for logo, toolbar, etc, and AppSidebar for the vertical nav.

This way we keep index pretty small, and have smaller components to move around.

Reply:
I broke the layout related code into its own component, didn't go so far as breaking out the header/sidebar as this gets into a grey area in terms of what we should recommend as a starting point, but what's there does generally follow the examples/demos so I'm feeling good about it. Let me know if you see any issues.